### PR TITLE
fix: add batch transform integration test

### PR DESCRIPTION
**Pull request by icywang86rui**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: test/integration/sagemaker/util.py [hunk 1]
```diff
@@ -60,11 +61,16 @@ def _production_variants(model_name, instance_type, accelerator_type):
 
     return production_variants
 
+
+def _test_bucket(region, boto_session):
+    account = boto_session.client('sts').get_caller_identity()['Account']
+    return f'sagemaker-{region}-{account}'
```
Comment (laurenyu) [inline test/integration/sagemaker/util.py]: what if the bucket doesn't exist?

File: test/integration/sagemaker/util.py [hunk 2]
```diff
@@ -124,6 +130,84 @@ def sagemaker_endpoint(sagemaker_client, model_name, instance_type, accelerator_
...
+        },
+        'TransformResources': {
+            'InstanceType': instance_type,
+            'InstanceCount': 1
+        }
+    }
+
+
+def _read_batch_output(boto_session, bucket, model_name):
+    s3 = boto_session.client('s3')
```
Comment (laurenyu) [inline test/integration/sagemaker/util.py]: add the region or else this won't work in GovCloud

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://3r4crpxqvk.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=596a9cbe-6096-4bbf-bbba-ac75ffbcc905%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: test/integration/sagemaker/util.py (hunk 1 above)
Comment (icywang86rui) [reply test/integration/sagemaker/util.py]: find_or_put_model_data will create the bucket

File: test/integration/sagemaker/util.py (hunk 2 above)
Comment (icywang86rui) [reply test/integration/sagemaker/util.py]: will do

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://3r4crpxqvk.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=cb5ff0ed-3ae4-4330-ae77-15cd574e390d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**